js-exercises/all-promises/allPromises.js

```diff
@@ -0,0 +1,13 @@
+const allPromises = async args => {
```

Comment:
If args is single promise, and not an array.
This program will return array, which it shouldn't.